src/main/scala/cromwell/binding/command/ParameterCommandPart.scala

```diff
@@ -13,7 +13,11 @@ case class ParameterCommandPart(wdlType: WdlType, name: String) extends CommandP
     if (wdlType != paramValue.wdlType) {
       throw new UnsupportedOperationException(s"Incompatible type for $name: need a $wdlType, got a ${paramValue.wdlType}")
     }
+
     /* TODO: this also is a gross over-simplification */
-    paramValue.value.toString
+    paramValue match {
```

Comment:
`asString` gets complicated for Arrays and Maps.  For example, there are at least a few ways in which an Array can be 'stringified'.  Primitives are trivially turned into strings.  I think we need to give more thought about how to implement stringification for compound types